api/src/main/kotlin/edu/wgu/osmt/richskill/RichSkillController.kt

```diff
@@ -66,10 +65,10 @@ class RichSkillController @Autowired constructor(
 
     val keywordDao = KeywordDao.Companion
 
-    override val allPaginatedPath: String = RoutePaths.SKILLS_LIST
+    override val allPaginatedPath: String = RoutePaths.Latest.SKILLS_LIST
     override val sortOrderCompanion = SkillSortEnum.Companion
 
-    @GetMapping(RoutePaths.SKILLS_LIST, produces = [MediaType.APPLICATION_JSON_VALUE])
+    @GetMapping(path = [RoutePaths.Latest.SKILLS_LIST], produces = [MediaType.APPLICATION_JSON_VALUE])
```

Comment:
I thinks this should be v2, `allPaginatedWithFilters` in v3 can do the same as allPaginated, so I think to have `allPaginated` and `allPaginatedWithFilters` in v3 is not needed.

Reply:
Yeah, we leave it as v2. now